protocol/message_handler.go

```diff
@@ -209,6 +209,8 @@ func (m *MessageHandler) HandleSyncInstallationContact(state *ReceivedMessageSta
 		}
 		contact.Photo = message.ProfileImage
 		contact.LastUpdated = message.Clock
+		contact.Nickname = message.Nickname
```

Comment:
Could you also update the tests (there should be one in `messenger_installations_test.go`) in order to make sure the nickname is also synced?